builtin/credential/aws/path_config_client.go

```diff
@@ -85,6 +87,12 @@ func (b *backend) pathConfigClient() *framework.Path {
 				Default:     aws.UseServiceDefaultRetries,
 				Description: "Maximum number of retries for recoverable exceptions of AWS APIs",
 			},
+
+			"role_arn": {
+				Type:        framework.TypeString,
+				Default:     "", // TODO: do we need to set the default here?
```

Comment:
Looks like the precedent is to set a default for the fields, so I would stick to setting a default for the scope of this PR. I think we can remove the `TODO` here, and can reconsider as needed in a future PR